app/parser/v0_0_1/schema_parser.py

```diff
@@ -0,0 +1,138 @@
+from app.parser.schema_parser import SchemaParser as AbstractSchemaParser
```

Comment:
Why not rename the SchemaParser class as AbstractSchemaParser? This confuses me a bit as we now have two SchemaParsers classes?